source/game_sa/PedClothesDesc.h

```diff
@@ -31,6 +31,15 @@ class CPedClothesDesc {
     bool HasVisibleTattoo();
     void SetTextureAndModel(uint32 texture, uint32 model, eClothesTexturePart texturePart);
     void SetTextureAndModel(const char* textureName, const char* modelName, eClothesTexturePart texturePart);
+
+    // NOTSA
+    bool IsWearingModel(const char* model, std::optional<eClothesModelPart> modelPart = {}) const {
+        if (modelPart.has_value()) {
+            return m_anModelKeys[+*modelPart] == CKeyGen::GetUppercaseKey(model);
+        } else {
+            return rng::contains(m_anModelKeys, CKeyGen::GetUppercaseKey(model));
+        }
```

Comment:
just use a ternary?

Reply:
too long for a one liner

Reply:
nah i meant my style like
```suggestion
        return modelPart.has_value()
            ? m_anModelKeys[+*modelPart] == CKeyGen::GetUppercaseKey(model)
            : rng::contains(m_anModelKeys, CKeyGen::GetUppercaseKey(model));
```

Reply:
i mean ok i guess